libraries/botbuilder-ai/src/qnaMaker.ts

```diff
@@ -464,18 +464,19 @@ export class QnAMaker implements QnAMakerTelemetryClient {
 
         this.validateOptions(queryOptions);
 
-        const qnaResult: any = await request({
-            url: url,
+        const qnaResult: any = await request( url, {
             method: 'POST',
             headers: headers,
             timeout: queryOptions.timeout,
-            json: {
+            body: JSON.stringify( {
                 question: question,
                 ...queryOptions
-            }
+            })
         });
-
-        return this.formatQnaResult(qnaResult);
+        
+        const qnaResultJson: any = await qnaResult.json();
+        
+        return this.formatQnaResult(qnaResultJson);
```

Comment:
Can we merged both lines (477-479) into a single one,  with something like this:

```
return this.formatQnaResult(await qnaResult.json());
```

Reply:
I have tried that approach, but it throws an exception.